pkg/timescale/write.go

```diff
@@ -112,35 +147,41 @@ func contains(s []string, e string) bool {
 }
 
 func insertRowsTimescale(rowsArray []TimescaleRows, config *config.OutputTimescale) error {
-	var c, parseErr = pgx.ParseConnectionString(config.Connection)
-	if parseErr != nil {
-		return parseErr
-	}
 
-	conn, connErr := pgx.Connect(c)
+	ctx := context.TODO()
```

Comment:
Is there a reason you are using `context.TODO()` instead of `context.Background`? IMHO, background makes more sense here.

Reply:
Since these two are quite similiar I agree that using Background makes more sense.